### PR TITLE
Add type for topologies

### PR DESCRIPTION
**Pull request by zmaslova**

### CODE REVIEW

File: tools/downloader/list_topologies.yml [hunk 1]
```diff
@@ -13,11 +13,9 @@
 # limitations under the License.
 
 topologies:
-#
-# PUBLIC TOPOLOGIES
-#
   - name: "densenet-121"
     description: "DenseNet-121 (https://github.com/shicai/DenseNet-Caffe). The following Caffe patch must be used to be able to convert this model: https://github.com/BVLC/caffe/pull/3057/files"
+    type: "classification"
```
Comment (IRDonch) [inline tools/downloader/list_topologies.yml]: Could you rename the field to `task_type`? I think that would be a better description of it.

File: tools/downloader/list_topologies.yml [hunk 2]
```diff
@@ -1533,6 +1590,7 @@ topologies:
       Head pose estimation network based on simple, handmade CNN architecture. Angle
       regression layers are convolutions + ReLU + batch norm + fully connected with
       one output.
+    type: "headpose"
```
Comment (IRDonch) [inline tools/downloader/list_topologies.yml]: I think this ought to be called `head_pose_estimation` for consistency with others (especially `human_pose_estimation`).

File: tools/downloader/list_topologies.yml [hunk 3]
```diff
@@ -826,6 +854,7 @@ topologies:
 
   - name: "license-plate-recognition-barrier-0007"
     description: "TensorFlow LPRNet"
+    type: "ocr"
```
Comment (IRDonch) [inline tools/downloader/list_topologies.yml]: I think that this should be called `optical_character_recognition`, for consistency.

File: tools/downloader/list_topologies.yml (hunk 1 above)
Comment (zmaslova) [reply tools/downloader/list_topologies.yml]: I copied it from our internal repo, but this is better field name here. Renamed

File: tools/downloader/list_topologies.yml (hunk 2 above)
Comment (zmaslova) [reply tools/downloader/list_topologies.yml]: done

File: tools/downloader/list_topologies.yml (hunk 3 above)
Comment (zmaslova) [reply tools/downloader/list_topologies.yml]: done

**Comment by kchechil** [comment]
Jenkins please retry a build

**Pull request merged**